### DISCUSSION TITLE
Taking a development meeting

### DISCUSSION THREAD

**Discussion by rusiruavb** [ms-club-sliit/msclub-backend#252] [General]
Hello @ms-club-sliit/dev-team 👋🏼,
It has been a while since we haven’t got a development meeting to discuss the progress of ongoing projects and share new ideas 💡. I think once the final examinations are done, we can take a meeting. Comment your ideas that need to be discussed at that meeting 😊.

**Comment by Chanuth-silva10** [comment]
I think What we did during this month and what is the improvement our project and our coding life. Thank you Rusiru aiya.
________________________________
From: Rusiru Abhisheak ***@***.***>
Sent: Saturday, May 21, 2022 10:00 PM
To: ms-club-sliit/msclub-backend ***@***.***>
Cc: De Silva C.M.K it20074418 ***@***.***>; Team mention ***@***.***>
Subject: [ms-club-sliit/msclub-backend] Taking a development meeting (Discussion #252)

[EXTERNAL EMAIL] This email has been received from an external source – please review before actioning, clicking on links, or opening attachments.

Hello @ms-club-sliit/dev-team<https://github.com/orgs/ms-club-sliit/teams/dev-team> 👋🏼,
It has been a while since we haven’t got a development meeting to discuss the progress of ongoing projects and share new ideas 💡. I think once the final examinations are done, we can take a meeting. Comment your ideas that need to be discussed at that meeting 😊.

—
Reply to this email directly, view it on GitHub<https://github.com/ms-club-sliit/msclub-backend/discussions/252>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATGQ7ORXW4MVZMF5CBCPVODVLEFQTANCNFSM5WSBJYVA>.
You are receiving this because you are on a team that was mentioned.Message ID: ***@***.***>

Disclaimer : This email and any attachment transmitted herewith are confidential and is intended solely for the use of the individual or entity to which they are addressed and may contain information that is privileged or otherwise protected from disclosure. If you are not the intended recipient, you are hereby notified that disclosing, copying, distributing, or taking any action in reliance on this email and the information it contains is strictly prohibited. If you have received this email in error, please notify the sender immediately by reply email and discard all of its contents by deleting this email and the attachment, if any, from your system.

Disclaimer : This email and any attachment transmitted herewith are confidential and is intended solely for the use of the individual or entity to which they are addressed and may contain information that is privileged or otherwise protected from disclosure. If you are not the intended recipient, you are hereby notified that disclosing, copying, distributing, or taking any action in reliance on this email and the information it contains is strictly prohibited. If you have received this email in error, please notify the sender immediately by reply email and discard all of its contents by deleting this email and the attachment, if any, from your system.
